index.js

```diff
@@ -0,0 +1,70 @@
+import Book from './modules/book.js'; //eslint-disable-line
+import * as luxon from './modules/luxon.js'; //eslint-disable-line
+
+const bookObj = new Book();
+
+const addBookButton = document.getElementById('addBook');
+
+addBookButton.addEventListener('click', (event) => {
+  event.preventDefault();
+  addBook(); //eslint-disable-line
+
+  const form = document.getElementById('book-add')
+  form.reset();
+});
+
+const generateListOfBooks = (books) => {
+  let items = [];
+
+  for (const [i, book] of books.entries()) { //eslint-disable-line
```

Comment:
- [ ] If localStorage has not been created (you can test this by deleting your own localStorage in your browser for this project), this happens when trying to add a new book (tested on both Firefox and Chrome after cleaning the local storage):

![image](https://user-images.githubusercontent.com/57421823/161953818-74c345ae-eb4c-40c2-8b5f-9e77f98a8859.png)

Please, kindly correct this so new people that try your app can add books.

## Explanation

This is happening because `books` argument is not being visible inside the for loop.  Therefore, the browser reports "books is undefined". 

## Hints

Just create a new variable and make it equal to the `books` argument of the function.

I managed to make the project work like this:

![image](https://user-images.githubusercontent.com/57421823/161954958-c11be886-0183-4d91-9e22-3802c93f43ef.png)

Now that you know the bug, and at least one solution, please go ahead and correct it! 💪